proto/internal/temporal/server/api/matchingservice/v1/request_response.proto

```diff
@@ -215,9 +215,12 @@ message GetWorkerBuildIdCompatibilityResponse {
 
 message GetTaskQueueUserDataRequest {
     string namespace_id = 1;
-    // The task queue to fetch data from. Is always considered as a workflow queue, since root
-    // workflow queues own the metadata.
+    // The task queue to fetch data from. The task queue is always considered as a normal
+    // queue, since sticky queues have no user data.
     string task_queue = 2;
+    // Normally task queue type should always be TASK_QUEUE_TYPE_WORKFLOW here, but querying
+    // activity task queues is useful for testing.
+    temporal.api.enums.v1.TaskQueueType task_queue_type = 5;
```

Comment:
@dnr this came from your change

Reply:
It's not that it's testing that specifically but it's checking propagation to activity task queues so that it can correctly test _other_ things. I tried to find another way and I couldn't. It's not even about the routing (which would be a problem if the tests used > 1 matching node), it's getting it to call the activity tqm.

But on the other hand, in the end I had to add a sleep for sticky queues anyway since there's no hope in enumerating those (without tons more invasive work). So... I dunno, I can undo this and things will work fine